sr_robot_launch/launch/sr_ur_arm_hand.launch

```diff
@@ -30,6 +30,8 @@
   <arg name="robot_state_pub_frequency" default="250"/>
   <!-- The joint state publisher publishing frequency  -->
   <arg name="joint_state_pub_frequency" default="125"/>
+  <!-- Logging wear data  -->
+  <arg name="wear_logging" default="true"/>
```

Comment:
I think we should set it to false for now, and enable it for hands that we want to record data for. we can discuss in show and tell